canonical-web-and-design/ubuntu.com
# Update robotics page

## Done

Updated /robotics

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/robotics
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check that the page matches [the copy doc](https://docs.google.com/document/d/1JNFk2C0shH3DapKuJM8X0QfAiD1RzHvXoydn8P9mEMg/edit?ts=5e20ceec#)

**This PR relies on #7105 being merged first**

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2572
